src/core/transaction.zig

```diff
@@ -170,19 +170,17 @@ pub const MessageAddressTableLookup = struct {
 };
 
 pub const Transaction = struct {
-    signatures: []Signature,
+    signatures: []const Signature,
     message: Message,
 
-    pub const @"!bincode-config:signatures" = ShortVecConfig(Signature);
+    pub const @"!bincode-config:signatures" = shortVecConfig([]const Signature);
 
     pub const MAX_BYTES: usize = 1232;
 
-    pub fn default() Transaction {
-        return Transaction{
-            .signatures = &[_]Signature{},
-            .message = Message.default(),
-        };
-    }
+    pub const empty: Transaction = .{
```

Comment:
Should this be `EMPTY`?

Reply:
0ffd7857944c8b15ebbd911fef814f071b8680c6